pkg/virt-launcher/virtwrap/network/bindings_test.go

```diff
@@ -0,0 +1,164 @@
+package network
+
+import (
+	"fmt"
+	"net"
+	"runtime"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/golang/mock/gomock"
+	"github.com/vishvananda/netlink"
+
+	kubevirtv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/pkg/network/cache"
+	"kubevirt.io/kubevirt/pkg/network/cache/fake"
+	netdriver "kubevirt.io/kubevirt/pkg/network/driver"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+var _ = Describe("BindingMechanism", func() {
```

Comment:
Where are the tests for bridge and masquerade bindings? This test file looks very partial.

Reply:
This PR just move the test we have, we can complete the test coverage at follow up PR but the task is about refactoring not code coverage.

Reply:
We have tests for masquerade and bridge binding. They were left in the the `podinterface_test.go` file.